src/features/components/molecules/protected-route/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PageNotFound from 'features/pages/404-error'
+import InitialLoader from 'components/elements/dot-loader'
+
+interface ProtectedRouteProps {
+    condition: boolean
```

Comment:
Can you make this boolean name as `is_condition`

Reply:
`show` `is_visible`

Reply:
updated with is_page_available @prince-deriv